rollup/internal/controller/sender/sender.go

```diff
@@ -369,10 +371,15 @@ func (s *Sender) resubmitTransaction(feeData *FeeData, auth *bind.TransactOpts,
 		if gasPrice.Cmp(maxGasPrice) > 0 {
 			gasPrice = maxGasPrice
 		}
-		feeData.gasPrice = gasPrice
 
-		txInfo["original_gas_price"] = originalGasPrice
-		txInfo["adjusted_gas_price"] = gasPrice
+		if originalGasPrice.Cmp(gasPrice) <= 0 {
```

Comment:
should it be `==`?

Reply:
changed in [4296d38](https://github.com/scroll-tech/scroll/pull/1058/commits/4296d38eb58699a7a9b397a68f543b3219de2b66)